src/main/java/net/snowflake/ingest/utils/Utils.java

```diff
@@ -88,26 +90,25 @@ private static Provider instantiateSecurityProvider() {
     }
   }
 
-  /**
-   * Assert when the String is null or Empty
-   *
-   * @param name
-   * @param value
-   * @throws SFException
-   */
+  /** Assert when the String is null or Empty */
   public static void assertStringNotNullOrEmpty(String name, String value) throws SFException {
     if (isNullOrEmpty(value)) {
       throw new SFException(ErrorCode.NULL_OR_EMPTY_STRING, name);
     }
   }
 
-  /**
-   * Assert when the value is null
-   *
-   * @param name
-   * @param value
-   * @throws SFException
-   */
+  /** Assert when all the Strings are either null or Empty */
+  public static void assertStringsNotNullOrEmpty(String name, Collection<String> values)
```

Comment:
nit: when I first saw this method, I assumed this was asserting that all the strings in the collection should be not null, maybe we can rename it to something like assertAtLeastOneStringNotNullOrEmpty to be extra clear?